App/UI/MessageHandler.cs

```diff
@@ -0,0 +1,138 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using App.Players;
+
+namespace App.UI
+{
+    public static class MessageHandler
+    {
+        public static void PrintGreeting()
+        {
+            Console.WriteLine("Welcome to Tic-Tac-Toe!");
+        }
+
+        public static void PrintGameModes()
+        {
+            Console.WriteLine("(1) Play against a friend");
+            // Console.WriteLine("(2) Play against an easy competitor");
+            // Console.WriteLine("(3) Play against a super computer");
+            Console.WriteLine("\n");
+        }
+
+        public static void PrintRequestToChooseGameMode()
+        {
+            Console.Write("Choose from one of the above options: ");
+        }
+
+        public static void PrintRequestToChooseGameModeAfterInvalidInput()
+        {
+            Console.Write("Invalid option. Choose again from options 1-3: ");
```

Comment:
This should match up with current code functionality: i.e. shouldn't reference other modes that don't exist yet and are commented out on lines 18, 19